pmd-apex/src/main/resources/category/apex/bestpractices.xml

```diff
@@ -168,4 +168,47 @@ trigger Accounts on Account (before insert, before update, before delete, after
 ]]>
         </example>
     </rule>
+
+    <rule name="DebugsShouldUseLoggingLevel"
+          since="6.17.0"
+          language="apex"
+          message="Calls to System.debug should specify a logging level."
+          class="net.sourceforge.pmd.lang.apex.rule.ApexXPathRule"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_apex_bestpractices.html#debugsshoulduselogginglevel">
+       <description>
+The first parameter of System.debug, when using the signature with two parameters, is a LoggingLevel enum.
+
+Having the Logging Level specified provides a cleaner log, and improves readability of it.
+       </description>
+       <priority>5</priority>
+       <properties>
+          <property name="strictMode" type="Boolean" value="false" description="If true, mark statements that use the DEBUG enum of LoggingLevel."/>
+          <property name="version" value="2.0"/>
+          <property name="xpath">
+             <value>
+<![CDATA[
+//MethodCallExpression
+  [count(ReferenceExpression[lower-case(@Image)='system']) = 1]
```

Comment:
do we actually need the count? can't this simply be:

```suggestion
  [ReferenceExpression[lower-case(@Image)='system']]
```

Reply:
@jsotuyod it think it needs to be `//MethodCallExpression[@FullMethodName='System.debug'][count(*)=2]`
because

1. we need to find System.debug(...) calls (Lowercase should also be discovered. Apex is case insensitive
2. We need to ignore System.debug(LoggingLevel.*, logText) because people put them on purpose. 
3. We should just flag System.debug(string);